src/app/modules/applications/pages/applications-page/applications-page.component.html

```diff
@@ -35,3 +33,38 @@
     </div>
   </div>
 </div>
+
+<ng-template #wrapper>
+  <div
+    tippy
+    [tippyOptions]="{
+      content: 'Deployment config required',
+      arrow: false,
+      theme: 'basic',
+      placement: 'bottom'
+    }"
+  >
+    <button
```

Comment:
DRY
You have already had button element as ng-template, reuse it
Use context to change template's behaviour